region/maridia/inner-pink/Draygon's Room.json

```diff
@@ -302,10 +302,15 @@
       "name": "Microwave Fight",
       "requires": [
         "h_canNavigateUnderwater",
-        "h_canBreakThreeDraygonTurrets",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          "h_canBreakThreeDraygonTurrets"
+        ]},
         "canXRayWaitForIFrames",
+        "Charge",
         "Plasma",
         {"or": [
+          "canTrickyDodgeEnemies",
```

Comment:
1 rounding with only plasma is bordering extreme, but you feel that its your own fault for not getting it.